subsys/bluetooth/controller/Kconfig

```
@@ -147,15 +151,42 @@ config BT_CTLR_HCI
 
 comment "BLE Controller configuration"
 
+config BT_CTLR_ENTROPY
+	bool "Random number generation in Controller"
+	depends on BT_CTLR_ENTROPY_SUPPORT
+	select ENTROPY_GENERATOR
+	default y
+	help
+	  Use random number generation provided by the Controller.
```

```suggestion
	  Use random number generation selected by the Controller.
```
?
(isn't just the selection that is left to the controller?